lower/llpcSpirvLowerResourceCollect.h

```diff
@@ -42,14 +42,41 @@ namespace Llpc
 struct DescriptorBinding;
 struct ResourceUsage;
 
+// Compact ResourceNodeData into an uint64 key
+union ResourceNodeDataKey
+{
+    struct
+    {
+        uint64_t set        :   16;  // Resource set
+        uint64_t binding    :   16;  // Resource binding
+        uint64_t arraySize  :   16;  // Resource array size
+        uint64_t reserved   :   16;
+    } nodeId;
```

Comment:
nodeId -> value
I think it is better to call the structure "value" for this "key".